new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -148,6 +146,39 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
                 .name(serviceAccountName))));
     logger.info("Created service account: {0}", serviceAccountName);
 
+    // get Operator image name
+    String operatorImage = getOperatorImageName();
+    assertFalse(operatorImage.isEmpty(), "Operator image name can not be empty");
+    logger.info("Operator image name {0}", operatorImage);
+
+    // Create docker registry secret in the operator namespace to pull the image from repository
+    if (System.getenv("REPO_REGISTRY") != null && System.getenv("REPO_USERNAME") != null
+        && System.getenv("REPO_PASSWORD") != null && System.getenv("REPO_EMAIL") != null) {
+      repoRegistry = System.getenv("REPO_REGISTRY");
+      repoUserName = System.getenv("REPO_USERNAME");
+      repoPassword = System.getenv("REPO_PASSWORD");
+      repoEmail = System.getenv("REPO_EMAIL");
+    }
+    logger.info("Creating docker registry secret in namespace {0}", opNamespace);
+    JsonObject dockerConfigJsonObject = createDockerConfigJson(
+        repoUserName, repoPassword, repoEmail, repoRegistry);
+    dockerConfigJson = dockerConfigJsonObject.toString();
+
+    // Create the V1Secret configuration
+    V1Secret repoSecret = new V1Secret()
+        .metadata(new V1ObjectMeta()
+            .name(repoSecretName)
+            .namespace(opNamespace))
+        .type("kubernetes.io/dockerconfigjson")
+        .putDataItem(".dockerconfigjson", dockerConfigJson.getBytes());
+
+    boolean secretCreated = assertDoesNotThrow(() -> createSecret(repoSecret),
+        String.format("createSecret failed for %s", repoSecretName));
+    assertTrue(secretCreated, String.format("createSecret failed while creating secret %s", repoSecretName));
```

Comment:
please include the namespace in the message too

Reply:
done